### DISCUSSION TITLE
Differences in cell numbers before and after SCTransform/IntegrateData

### DISCUSSION THREAD

**Discussion by gvogler** [satijalab/seurat#5863] [Q&A]
I am puzzled by the following observation: When I run ```SCTransform``` on my samples (here: 2) I get the following:
```
$`10Xtwi24_GD`
An object of class Seurat 
34631 features across 2306 samples within 2 assays 
Active assay: SCT (10697 features, 3000 variable features)
 1 other assay present: RNA

$`10Xtwi24_GD_4_6`
An object of class Seurat 
34449 features across 2091 samples within 2 assays 
Active assay: SCT (10515 features, 3000 variable features)
 1 other assay present: RNA
```
 
After integration (```SelectIntegrationFeatures -> ... -> IntegrateData``` steps) the output of ```table(seurat$orig.ident)``` produces these numbers:

```
10Xtwi24_GD   10Xtwi24_GD_4_6
2830          2850
```

These are more cells than enter the analysis?! I do not quite understand the data integration step, specifically how cell numbers are changed here. Any pointers that would help me understand what is going on would be greatly appreciated!

P.S. Using different starting numbers (less stringent filtering during ```subset``), I get similar results:

```
$`10Xtwi24_GD`
An object of class Seurat 
34777 features across 3153 samples within 2 assays 
Active assay: SCT (10843 features, 3000 variable features)
 1 other assay present: RNA

$`10Xtwi24_GD_4_6`
An object of class Seurat 
35148 features across 6439 samples within 2 assays 
Active assay: SCT (11214 features, 3000 variable features)
 1 other assay present: RNA
```
```
10Xtwi24_GD   10Xtwi24_GD_4_6  
5428          5440
```

**Comment by PilanEli** [comment]
Hi,

The SC Transform function must be applied only between datasets or also when we integrate conditions from the same dataset. 
Example: I'm working with the Introduction to scRNA-seq integration vignette in a dataset with 2 conditions (control and tumor). In this case should I use the SCTransform function?
